pkgs/top-level/all-packages.nix

```diff
@@ -9236,10 +9236,9 @@ with pkgs;
 
   minissdpd = callPackage ../tools/networking/minissdpd { };
 
-  inherit (callPackage ../tools/networking/miniupnpc
-            { inherit (darwin) cctools; })
-    miniupnpc_1 miniupnpc_2;
-  miniupnpc = miniupnpc_1;
+  miniupnpc = callPackage ../tools/networking/miniupnpc {
+    inherit (darwin) cctools;
+  };
```

Comment:
Should there be a throws here for miniupnpc_1 explaining it's eol and unmaintained instead of just removing it?

Reply:
Good catch, noted removal to `aliases.nix` in d6604db49e00d51c0a3aeb0949463bc997c14da2